src/Utilities/Colors/Color.php

```diff
@@ -199,7 +199,7 @@ private static function convertHSLtoRGB(float $h, float $s, float $l): array
         $s = min(max($s, 0), 1);
         $l = min(max($l, 0), 1);
 
-        if ($s == 0) {
+        if ($s === 0) {
```

Comment:
Since PHP is weird, this does not result in the same behavior. Tested on PHP 7.4.7 (CLI):

```
> php
<?php
var_dump(0.0 === 0);
^Z
Standard input code:2:
bool(false)
```

In other words, float `0.0` compares to int `0` as `false` (at least in 7.4). I suggest the following:

```suggestion
        if ((float) $s === 0.0) {
```

What do you think?